<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mbCoreDialogSystemSettings</class>
 <widget class="QDialog" name="mbCoreDialogSystemSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pgView">
      <attribute name="title">
       <string>View</string>
      </attribute>
      <widget class="QCheckBox" name="chbUseNameWithSettings">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>217</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Use Port/Device names with settings</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chbUseNameWithSettings_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>217</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Use Port/Device names with settings</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pgLog">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="grLogFlags">
         <property name="title">
          <string>Log Flags</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="chbLogError">
            <property name="text">
             <string>Error</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbLogWarning">
            <property name="text">
             <string>Warning</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbLogInfo">
            <property name="text">
             <string>Info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbLogTxRx">
            <property name="text">
             <string>TxRx</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chbUseTimestamp">
         <property name="text">
          <string>Use timestamp</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Format</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lnFormat"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mbCoreDialogSystemSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mbCoreDialogSystemSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
